### PR TITLE
wasm-bindgen removed

### PR DESCRIPTION
**Pull request by szagi3891**

### CODE REVIEW

File: crates/vertigo-browserdriver/src/utils/mod.rs [hunk 1]
```diff
@@ -1,3 +1,4 @@
 pub mod callback_manager;
 pub mod future;
 pub mod json;
+pub mod futures_spawn;
```
Comment (sfisol) [inline crates/vertigo-browserdriver/src/utils/mod.rs:4]: add newline

File: crates/vertigo-browserdriver/src_js/index.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { WasmModule } from "./wasm_module";
+
+export const runModule = async (wasmBinPath: string) => {
+    console.info(`Wasm module: "${wasmBinPath}" -> start`);
+    const wasmModule = await WasmModule.create(wasmBinPath);
+    console.info(`Wasm module: "${wasmBinPath}" -> initialized`);
+    wasmModule.start_application();
+    console.info(`Wasm module: "${wasmBinPath}" -> launched start_application function`);
+};
```
Comment (sfisol) [inline crates/vertigo-browserdriver/src_js/index.ts:9]: add newline

File: crates/vertigo-browserdriver/src/wasm_run.js [hunk 3]
```diff
@@ -0,0 +1,882 @@
...
+// type KeydownCallbackType = (
+//     // dom_id: BigInt | null,
+//     dom_id: BigInt,         //0 - null
+//     // key: string,
+//     // code: string,
+//     alt_key: boolean,
+//     ctrl_key: boolean,
+//     shift_key: boolean,
+//     meta_key: boolean
+// ) => boolean;
```
Comment (sfisol) [inline crates/vertigo-browserdriver/src/wasm_run.js:75]: Can this commented code be removed completly?

File: crates/vertigo-browserdriver/src_js/init_wasmbindgen.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+        } else {
+            return instance;
+        }
+    }
+}
+
+async function init(input) {
+    if (typeof input === 'undefined') {
+        input = new URL('vertigo_demo_bg.wasm', import.meta.url);
```
Comment (sfisol) [inline crates/vertigo-browserdriver/src_js/init_wasmbindgen.js:38]: Hardcoded demo filename, but we're in vertigo browserdriver

File: crates/vertigo-browserdriver/src_js/wasm_module.ts [hunk 5]
```diff
@@ -0,0 +1,247 @@
...
+        dom_id: BigInt                                                                          //0 - null
+    ) => void;
+    dom_keydown: (                                                                              //Parametry dla rust na stosie: key: string, code: string,
+        dom_id: BigInt,                                                                         // 0 - null
+        alt_key: number,                                                                        // 0 - false, >0 - true
+        ctrl_key: number,                                                                       // 0 - false, >0 - true
+        shift_key: number,                                                                      // 0 - false, >0 - true
+        meta_key: number                                                                        // 0 - false, >0 - true
+    ) => number;                                                                                // 0 - false, >0 - true
+    dom_oninput: (dom_id: BigInt) => void,                                                      //Parametr dla rust na stosie: text: string
```
Comment (sfisol) [inline crates/vertigo-browserdriver/src_js/wasm_module.ts:97]: Translate comments to english

File: crates/vertigo-macro/Cargo.toml [hunk 6]
```diff
@@ -15,5 +15,6 @@ pest = "2.1.3"
 pest_derive = "2.1.0"
 proc-macro2 = "1.0.24"
 proc-macro-error = "1.0.4"
-syn = "1.0.60"
+# syn = "1.0.60"
```
Comment (sfisol) [inline crates/vertigo-macro/Cargo.toml:18]: remove commented line

File: tsconfig.json [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+        "noUnusedLocals": true,
+        "noUnusedParameters": true,
+        "noUncheckedIndexedAccess": true,
+        "allowSyntheticDefaultImports": true,
+        "esModuleInterop": true,
+        "forceConsistentCasingInFileNames": true,
+        "baseUrl": ".",
+        "noEmit": true
+    }
+}
```
Comment (sfisol) [inline tsconfig.json:18]: add newline

**Review by sfisol — CHANGES_REQUESTED** [review]

File: Makefile.toml [hunk 8]
```diff
@@ -46,10 +48,6 @@ dependencies = [
 command = "cargo"
 args = [ "watch", "--watch", "./demo/app", "--watch", "crates", "--delay", "0.5", "-x", "make demo-start" ]
 
-[tasks.demo-clean]
-command = "rm"
-args = [ "-rf", "demo/build" ]
-
```
[the remark this answers is not in the document]
Comment (szagi3891) [reply Makefile.toml:52]: This is already included in the build task

File: crates/vertigo-browserdriver/src_js/init_wasmbindgen.js (hunk 4 above)
Comment (szagi3891) [reply crates/vertigo-browserdriver/src_js/init_wasmbindgen.js:38]: This is the file for analysis and the nicer things need to be copied into our solution

File: demo/app/index.html [hunk 9]
```diff
@@ -3,8 +3,8 @@
     <head>
         <meta charset="utf-8"/>
         <script type="module">
-            import init from "./demo.js";
-            init("./demo_bg.wasm");
+            import { runModule } from "/wasm_run.js";
```
[the remark this answers is not in the document]
Comment (szagi3891) [reply demo/app/index.html]: done

File: crates/vertigo-browserdriver/build.rs [hunk 10]
```diff
@@ -0,0 +1,16 @@
+use std::fs;
+use std::env;
+use std::path::PathBuf;
+
+fn main() {
+    println!("cargo:rerun-if-changed=glue.js");
```
[the remark this answers is not in the document]
Comment (szagi3891) [reply crates/vertigo-browserdriver/build.rs]: done

File: crates/vertigo-browserdriver/src/utils/mod.rs (hunk 1 above)
Comment (szagi3891) [reply crates/vertigo-browserdriver/src/utils/mod.rs]: done

File: crates/vertigo-browserdriver/src_js/wasm_module.ts (hunk 5 above)
Comment (szagi3891) [reply crates/vertigo-browserdriver/src_js/wasm_module.ts]: done

File: crates/vertigo-macro/Cargo.toml (hunk 6 above)
Comment (szagi3891) [reply crates/vertigo-macro/Cargo.toml]: done

File: tsconfig.json (hunk 7 above)
Comment (szagi3891) [reply tsconfig.json]: done

File: crates/vertigo-browserdriver/src/wasm_run.js (hunk 3 above)
Comment (szagi3891) [reply crates/vertigo-browserdriver/src/wasm_run.js:75]: done

File: crates/vertigo-browserdriver/src_js/index.ts (hunk 2 above)
Comment (szagi3891) [reply crates/vertigo-browserdriver/src_js/index.ts]: done

File: Makefile.toml (hunk 8 above)
[the remark this answers is not in the document]
Comment (sfisol) [reply Makefile.toml:52]: Now there is no possiblility to only clean the project without building it again.

File: crates/vertigo-browserdriver/src_js/init_wasmbindgen.js (hunk 4 above)
Comment (szagi3891) [reply crates/vertigo-browserdriver/src_js/init_wasmbindgen.js]: file deleted

File: Makefile.toml (hunk 8 above)
[the remark this answers is not in the document]
Comment (szagi3891) [reply Makefile.toml:52]: I have restored this task

**Review by sfisol — APPROVED** [review]

**Pull request merged**